db/db.go

```diff
@@ -0,0 +1,56 @@
+package db
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+
+	_ "github.com/mattn/go-sqlite3"
+
+	"github.com/uptrace/bun"
+	"github.com/uptrace/bun/dialect/sqlitedialect"
+	"github.com/uptrace/bun/extra/bundebug"
+)
+
+var DB *RootDB
+
+type RootDB = bun.DB
```

Comment:
why bother with this type alias?

Reply:
this is an omission - I didn't get rid of RootDB and just aliased it so don't need to refactor it.